InMobiAdsDemo/InMobiAdsDemo/Controllers/IMNativeViewController.m

```diff
@@ -0,0 +1,243 @@
+//
+//  IMStrandsViewController.m
+//  Ads Demo
+//
+//  Copyright © 2016 Inmobi. All rights reserved.
+//
+
+#import "IMNativeViewController.h"
+#import <InMobiSDK/InMobiSDK.h>
+#import "IMSimpleTableCell.h"
+#import "IMAppIds.h"
+
+#define kIMAdInsertionPosition 5
+@interface IMNativeViewController() <IMNativeDelegate,UITableViewDelegate,UITableViewDataSource>
+@property (nonatomic,strong) IMNative *native;
+@property (nonatomic) BOOL nativeStrandsLoaded,nativeStrandsRendered,nativeStrandsInserted;
+@property (nonatomic,strong) UITableView *tableView;
+@property (nonatomic,strong) NSMutableArray *tableData;
+@end
+@implementation IMNativeViewController
+- (void)viewDidLoad {
+    self.tableData = [[NSMutableArray alloc] init];
+    [self loadInitialData];
+    [self loadInitialData];
+    if(self.isStoryBoard){
+        self.native = [[IMNative alloc] initWithPlacementId:INMOBI_NATIVE_STORYBOARD];
+    }
+    else{
+        self.native = [[IMNative alloc] initWithPlacementId:INMOBI_NATIVE_MOVIEBOARD];
+    }
+    self.native.delegate = self;
+    [self.native load];
+    
+    self.tableView = [[UITableView alloc] initWithFrame:self.view.frame style:UITableViewStylePlain];
+    self.tableView.delegate = self;
+    self.tableView.dataSource = self;
+    [self.view addSubview:self.tableView];
+    [self.tableView addObserver:self
+                     forKeyPath:@"contentOffset"
+                        options:NSKeyValueObservingOptionNew | NSKeyValueObservingOptionOld
+                        context:nil];
+}
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+    return _tableData.count;
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+    if ([self isAdAtIndexPath:indexPath]) {
+        UITableViewCell *cell;
+        cell = [tableView dequeueReusableCellWithIdentifier:@"adIdentifier"];
+        if (cell == nil) {
+            cell = [[UITableViewCell alloc] initWithStyle:UITableViewCellStyleDefault reuseIdentifier:@"adIdentifier"];
+        }
+        IMNative *strands = [self.tableData objectAtIndex:indexPath.row];
+        [cell addSubview:[strands primaryViewOfWidth:cell.frame.size.width]];
+        return cell;
+    } else {
+        IMSimpleTableCell *cell = (IMSimpleTableCell *)[tableView dequeueReusableCellWithIdentifier:@"SimpleTableCell"];
+        cell = [tableView dequeueReusableCellWithIdentifier:@"SimpleTableCell"];
+        if (cell == nil) {
+            NSArray *nib = [[NSBundle mainBundle] loadNibNamed:@"SimpleTableCell" owner:self options:nil];
+            cell = [nib objectAtIndex:0];
+            id slide = [self.tableData objectAtIndex:indexPath.row];
+            SampleData *current_slide = slide;
+            cell.Title.text = current_slide.title;
```

Comment:
Please follow camelCasing with all the properties of SampleData

Reply:
done